android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.java

```diff
@@ -168,6 +190,11 @@ public void setEnabled(ButtonViewGroup view, boolean enabled) {
     view.setEnabled(enabled);
   }
 
+  @ReactProp(name = ViewProps.BORDER_RADIUS)
+  public void setBorderRadius(ButtonViewGroup view, float borderRadius) {
+    view.setBorderRadius(borderRadius);
```

Comment:
don't we need to set `mNeedBackgroundUpdate = true` here?

Reply:
yep, we need

Reply:
maybe not here but in the `setBorderRadius` method